scripts/ingests/bones_archive/ingest_bones_sources.py

```diff
@@ -32,24 +38,14 @@
 no_sources = 0
 inside_if = 0
 
-# Logger setup
-# This will stream all logger messages to the standard output and
-# apply formatting for that
-logger.propagate = False  # prevents duplicated logging messages
-LOGFORMAT = logging.Formatter(
-    "%(asctime)s %(levelname)s: %(message)s", datefmt="%m/%d/%Y %I:%M:%S%p"
-)
-ch = logging.StreamHandler(stream=sys.stdout)
-ch.setFormatter(LOGFORMAT)
-# To prevent duplicate handlers, only add if they haven't been set previously
-if len(logger.handlers) == 0:
-    logger.addHandler(ch)
-logger.setLevel(logging.INFO)
 
 DB_SAVE = False
 RECREATE_DB = True
 db = load_astrodb("SIMPLE.sqlite", recreatedb=RECREATE_DB, reference_tables=REFERENCE_TABLES)
 
+ingest_publication(db, bibcode="2018MNRAS.479.1383Z", reference="Zhan18.1352")
+ingest_publication(db, bibcode="2018MNRAS.480.5447Z", reference="Zhan18.2054")
+
```

Comment:
The `ingest_publications` function wanted to call these both `Zhan18` so I just ingested them by specifying the reference to be  Zhan18.1352 and Zhan18.2054.